docs/.vuepress/config.js

```diff
@@ -869,17 +869,69 @@ module.exports = {
     },
     '/zh/': {
       lang: 'zh-CN',
-      label: translate('language', 'zh-CN'),
-      title: translate('ethereum', 'zh-CN'),
+      label: translate('language'),
```

Comment:
This should be `label: translate('language', 'zh-CN'),`

Reply:
Please see my comment above.